.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "third_party/googletest"]
 	path = third_party/googletest
 	url = https://github.com/google/googletest
+[submodule "third_party/hipify_torch"]
```

Comment:
Do we need this? Has PyTorch already adopted this submodule?

Reply:
Yes, we are moving to hipify-torch repo, even for PyTorch this change is under review.